app/(main)/register/page.tsx

```diff
@@ -105,8 +107,10 @@ const Register = (): JSX.Element => {
     data: RegisterUserSchemaType,
   ): Promise<void> => {
     try {
+      setIsLoading(true);
       await registerAccount(data);
```

Comment:
Need to mock these 2 functions. 1 of these needs a setTimeout so you can test that the loading spinner is there. Once the setTimeout is done, it will run its course and setIsLoading will become false.

Reply:
If we go with this method, we don't need the custom setTimeout. 
https://github.com/LetsGetTechnical/gridiron-survivor/pull/454/files#diff-4c664975b58eb9a2b8fd8520303503caa5c63ea6e1c3f47e2682744b30c2623eR45